### PR TITLE
Update Login & SignUp screen

### PR DESCRIPTION
**Pull request by nehakumari2307**
The login screen is updated to include links for Forgot password, set a new password, and Registering new employer.

The SignUp screen for the admin to add a new account is updated with a note for newly added clients.

@ericfu666 Please take a pull once approved and add the page link for the reset password and set new password to the Login Screen

### CODE REVIEW

**Review by ericfu666 — APPROVED** [review]

File: src/pages/admin/AccountCreate.jsx [hunk 1]
```diff
@@ -41,8 +42,14 @@ export default function AccountCreate() {
       <div className="card max-w-md">
         <div className="card-body">
           <p className="mb-5 text-2xl">Create an Account</p>
-          {/* @tianchi @neha todo: may add an explantion for the sign up here */}
-          <p>@tianchi @neha We could add a explanation here</p>
+          <p>
+            <InfoIcon /> <strong>Note:</strong>
```
Comment (makersmelx) [inline src/pages/admin/AccountCreate.jsx:46]: use className `text-bold`

File: src/pages/SignIn.jsx [hunk 2]
```diff
@@ -80,26 +76,82 @@ export default function SignIn() {
...
+                  Click Here To Set Password For First Time
+                </Link>
+              }
+            </div>
+          </div>
+          <br />
+          <div className="text-sm">
+            <div className="flex justify-start gap-4">
+              <BadgeIcon fontSize="small" />
+              New Employer to RefugeeOne?{" "}
```
Comment (makersmelx) [inline src/pages/SignIn.jsx:128]: remove the blank

File: src/pages/SignIn.jsx [hunk 3]
```diff
@@ -80,26 +76,82 @@ export default function SignIn() {
...
+              <ArrowForwardIcon fontSize="medium" />
             </button>
           </form>
-          {/* todo */}
-          <p className="text-sm mt-5">Don&apos;t have email registered yet?</p>
-          <p className="text-sm font-bold">Connect with RefugeeOne team to get started</p>
+          <div>
+            <hr
+              style={{
+                background: "#666663",
```
Comment (makersmelx) [inline src/pages/SignIn.jsx:103]: what's this color?

File: src/pages/SignIn.jsx [hunk 4]
```diff
@@ -80,26 +76,82 @@ export default function SignIn() {
...
+          </div>
+          <br />
+          <div className="text-sm">
+            <div className="flex justify-start gap-4">
+              <EmailIcon fontSize="small" />
+              Don&apos;t have email registered yet with RefugeeOne?{" "}
+            </div>
+            <div className="flex space-around gap-4 text-sm">
+              {
+                <a
```
Comment (makersmelx) [inline src/pages/SignIn.jsx:146]: I think we should open this in a new tab

**Pull request merged**